jdaviz/configs/specviz2d/tests/test_parsers.py

```diff
@@ -89,6 +91,37 @@ def test_2d_parser_no_unit(specviz2d_helper, mos_spectrum2d):
     assert label_mouseover.icon == 'b'
 
 
+def test_2d_parser_hdulist_ext(tmp_path, specviz2d_helper, mos_spectrum2d_as_hdulist):

```

Comment:
This goes through a _similar_ logic route as Cami's problematic case but does not give the exact traceback, probably because I am trying to fake it. But I don't feel like downloading even more data for the test suite and none of the existing remote data tests are triggering Cami's error. 🤷‍♀️